tests/saitests/py3/switch.py

```diff
@@ -822,14 +822,38 @@ def sai_thrift_read_port_watermarks(client, port):
     return (queue_res, pg_shared_res, pg_headroom_res)
 
 
-def sai_thrift_read_pg_counters(client, port_id):
+def sai_thrift_read_pg_counters(client, target, port_id):
```

Comment:
sai_thrift_read_pg_counters is invoked in a variety of locations in other tests that would need to be updated.

Reply:
Updated.